frontend/src/logic/schedule-logic/shifts-info.logic.ts

```diff
@@ -49,12 +52,12 @@ export class ShiftsInfoLogic extends BaseSectionLogic implements ShiftsProvider
     return this.sectionData.length;
   }
 
-  public getWorkerShifts(): { [nurseName: string]: ShiftCode[] } {
-    return DataRowHelper.dataRowsAsValueDict<ShiftCode>(Object.values(this.shifts), false);
+  public get workers(): string[] {
+    return Object.keys(this.shifts);
   }
 
-  public getWorkers(): string[] {
-    return Object.keys(this.shifts);
+  public getWorkerShifts(): { [nurseName: string]: ShiftCode[] } {
+    return DataRowHelper.dataRowsAsValueDict<ShiftCode>(Object.values(this.shifts), false);
   }
```

Comment:
Zastanawiam się nad konwecją `get workerShifts`, czy `getWorkerShifts`?

Reply:
W większości miejsc zmieniłem na `get workerShifts`, ale w tym miejscu zostawiłem jako funkcję bo mi się wydaje, że w przyszłości możemy dodać więcej logiki do tej funkcji (np. `getWorkerShifts(fromDate, toDate)` ) i żeby potem nie było większego problemu z tym żeby to zmieniać. 
Co wy o tym sądzicie ?

Reply:
Na razie trzymajmy się tej jednej konwencji, a jeśli będzie to potrzebne to wtedy to faktycznie dostosujemy

Reply:
Zamieniłem na gettery